src/manifest.json

```diff
@@ -40,6 +40,13 @@
 				"vendor/@cliqz/adblocker-webextension-cosmetics/adblocker.umd.min.js",
 				"content_scripts/cosmetics.js"
 			]
+		},
+		{
+			"matches": ["*://www.google.com/*", "*://google.com/*"],
```

Comment:
what about `.de` and others?

Reply:
Yes, they are currently missing. Would like like that:

https://github.com/Rob--W/dont-track-me-google/blob/abffd08f6e0d6eacd42b3ff0ccda1b6838788949/manifest.json#L14

Reply:
Changed.